ru/verification.json

```
@@ -1,22 +1,22 @@
 {
-	"get_verified": "Пройти проверку",
-	"verification_form": "Форма подтверждения",
-	"verification_status": "Статус проверки",
+	"get_verified": "Пройти верификацию",
+	"verification_form": "Форма верификации",
+	"verification_status": "Статус верификации",
 	"cancel": "Отмена",
 	"continue": "Продолжать",
 	"profile": "Профиль",
-	"success_message": "успех",
+	"success_message": "Успешная подача заявки",
 	"error_message": "Что-то пошло не так, повторите попытку позже",
 	"requirement": {
-		"introduction": "Как художник Paras, вам необходимо пройти верификацию, чтобы получить доступ ко всем функциям Paras. Верификация дает дополнительную видимость на рынке. В настоящее время непроверенные NFT исполнителя не будут отображаться в ленте «рынка». Весь процесс занимает 3-4 дня.",
+		"introduction": "Как художник Paras, вы должны быть верифицированы, чтобы получить доступ к полным возможностям Paras. Верификация дает дополнительную видимость на торговой площадке. NFT непроверенного исполнителя не будут появляться в ленте "рынка". Весь процесс займет 3-4 дня.",
```

Hi @aufv there is a JSON error on this line
```suggestion
		"introduction": "Как художник Paras, вы должны быть верифицированы, чтобы получить доступ к полным возможностям Paras. Верификация дает дополнительную видимость на торговой площадке. NFT непроверенного исполнителя не будут появляться в ленте \"рынка\". Весь процесс займет 3-4 дня.",
```

---

@ahnafalfariza oops.  try it now